contracts/Super1155.sol

```diff
@@ -325,24 +255,17 @@ contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataUR
   /**
     Construct a new ERC-1155 item collection.
 
-    @param _owner The address of the administrator governing this collection.
     @param _name The name to assign to this item collection contract.
     @param _uri The metadata URI to perform later token ID substitution with.
     @param _proxyRegistryAddress The address of a proxy registry contract.
   */
-  constructor(address _owner, string memory _name, string memory _uri,
```

Comment:
Ditto. We like this pattern. If there's some issue with it that merits its removal let us know, but otherwise I'd like to keep it.